src/security/Handshake.cc

```diff
@@ -550,23 +551,26 @@ Security::HandshakeParser::ParseCertificate(const SBuf &raw, Security::CertPoint
     pCert = nullptr;
     debugs(83, 2, "TLS parsing is not supported without OpenSSL. " << raw);
```

Comment:
Please remove `debugs()`. This certificate-creation method cannot return a nil pointer! In your new code, the `#else` part exists only because we have to quiet compiler warnings until CertPointer declarations are fixed. This `#else` code should never be executed!

_Ideally_, I would wrap the whole method definition (but not the declaration) in `#ifdefs` but I am worried that some linkers might complain that the static method is never defined. If you think that my worries are unfounded (e.g., there are other undefined static methods in Squid already), then please remove `#else` and extend the `USE_OPENSSL` guards to the whole method definition instead.